### ISSUE TITLE
Unable to drag progress bar when rotated 90 degrees in iOS Safari

### ISSUE THREAD

**Issue by user080975** [videojs/video.js#8177]
### Description

Hi,

When the video container is rotated 90 degrees in iOS Safari, I'm unable to drag the progress bar up or down to progress the video. Is there a way to get the progress bar to work in iOS Safari when the page is rotated 90 degrees?

Thank You!

### Steps to reproduce

1. Create a regular VideoJS player
2. Rotate video container 90 degrees with CSS
3. Try to drag progress bar in iOS Safari

### What version of Video.js are you using?

8.0.4

### What browser(s) including version(s) does this occur with?

iOS Safari

### What OS(es) and version(s) does this occur with?

iOS

**Comment by gkatsev** [comment]
The progress bar needs to be configured so that it knows that it's vertical.

Something like:
```
videojs('my-video', {
  controlBar: {
    progressControl: {
      seekBar: {
        vertical: true
      }
    }
  }
});
```

**Comment by user080975** [comment]
Thank you for your reply! I tried this just now and I'm still unable to drag the seek bar. 

I also notice that the seek bar isn't actually progressing as the video plays when rotated 90 degrees. Is this what's causing the issue?

**Comment by user080975** [comment]
I made a very example here but adding the `transform: rotate(90deg);` CSS to the default video player, if you open this link and click the video to play, then try and drag the seek bar, nothing happens:

https://codepen.io/jcy080/pen/ZEMXLdM

**Comment by gkatsev** [comment]
Yeah, it definitely won't work without that configuration, however, a quick check and seems like it isn't making it work either, not sure what's wrong.

**Comment by user080975** [comment]
So normally that configuration should work?

**Comment by gkatsev** [comment]
I'd expect it to work. Also, your example is using Video.js 5 which is an incredibly old version.

**Comment by user080975** [comment]
I've changed the Video.js version to the latest one as well as added the parameters you mentioned:
https://codepen.io/jcy080/pen/ZEMXLdM

If you try dragging the seek bar in iOS Safari, you'll see that it's really inaccurate and often can't be properly dragged.

**Comment by user080975** [comment]
I notice that if you drag the seek bar left and right then it moves up and down correctly, so I'm assuming there needs to be some sort of orientation parameter that needs to be changed? From left -> right to up -> down?

**Comment by gkatsev** [comment]
> If you try dragging the seek bar in iOS Safari, you'll see that it's really inaccurate and often can't be properly dragged.

I think that's just because of the video. It doesn't have very precise seeking capabilities. Try a different source.

**Comment by user080975** [comment]
The video I used is the first demo one from the VideoJS official website and it drags fine when not rotated 90 degrees, so it's not an issue with the video right? 

If possible, could you please try the demo link here and see if you are able to drag vertically up and down on a mobile browser?
https://codepen.io/jcy080/pen/ZEMXLdM

**Comment by gkatsev** [comment]
Ok, cool, I guess it needs setting `vertical` on `progressControl` directly rather than on the seekBar. Also, when used in data-setup, it needs to be updated to be valid JSON by quoting all the properties.

This worked
```
  <video id="my_video_1" class="video-js vjs-default-skin" width="640px" height="267px" controls preload="none" poster='http://video-js.zencoder.com/oceans-clip.jpg' data-setup='{ "aspectRatio":"640:267", "controlBar": {
    "progressControl": {
      "vertical": true,
    }
  } }'>
```

**Comment by user080975** [comment]
I copied your code directly into the Codepen example but still couldn't get it to work unfortunately. If possible, would you be able to send me a working Codepen link so I can replicate it?

Thank You!
